CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ set(GENERATED_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/src/inc)
 
 function(clr_unknown_arch)
     if (WIN32)
-        message(FATAL_ERROR "Only AMD64 and I386 are supported")
+        message(FATAL_ERROR "Only AMD64, ARM64 and I386 are supported")
```

Comment:
Is I386 really supported in CoreCLR currently?  Otherwise we can unify these message()'s and drop the if()